dev-tools/audius-compose

```diff
@@ -565,7 +565,6 @@ def up(
                 f"--file={protocol_dir / 'dev-tools/compose/docker-compose.yml'}",
                 *profiles,
                 "up",
-                "--build",
```

Comment:
this requires everyone to use an `audius-compose build` + `audius-compose up` flow now right?

Reply:
I still need to prove this out, but my reading of https://docs.docker.com/compose/compose-file/build/#using-build-and-image suggests maybe docker compose is smart enough to build if there's a `build` config?

Reply:
my test locally and the CI seemed to work! I'm uncertain whether that means they're using the local image or if somehow it's finding images to pull but I don't _think_ those images exist and no `image` config is specified in the docker compose files so we _should_ be good!

Reply:
I think we need to keep `--build` if we want devving on `Dockerfile.dev` to work. If we want people to manually run `audius-compose build` before runs if they are touching anything that could impact the image, then we can omit. feels like we'll get into a situation where people are out of date though

Reply:
Yeah that's mostly what I'm concerned with, the images don't change often but it makes sense that a run command wouldn't build something

Reply:
@rickyrombo let's bring this back now right? Since it's divorced from the validator change now.

Reply:
Ray and I paired on this and got it working, but it required a split of the configs again

Reply:
Ah yeah jinx. I thought you already added it back in your commits, one sec

Reply:
Done